deploy/examples/crimson-asan-env-override.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
How about the following?
- Name this file `osd-env-override.yaml`
- Add comments at the top about how this configmap will be used. I don't think we need official documentation on this developer feature, just documenting here in the configmap seems sufficient.
- Add a simple comment below on why the ASAN_OPTIONS would be used in the example

Reply:
Reworked.